app/controllers/api/v1/messages_controller.rb

```diff
@@ -1,8 +1,11 @@
 module Api
   module V1
     class MessagesController < Api::V1::ApiController
+      include Pagy::Backend
       def index
-        @messages = policy_scope(conversation.messages).includes(:user)
+        messages = policy_scope(conversation.messages).includes(:user)
+        per_page = 5
```

Comment:
maybe it can be a param too, what do you think?
and also, what happens if the request does not receive this params ? can we test this scenario ?

Reply:
Done too.

Reply:
Its ok. would be nice to move the "5" to a const or a env var. 
Just a comment, another possibility  is check that the parameter is within a minimum and maximum range to avoid requests for an excessive number of items, for instance, one million items are requested

Reply:
Done!